subworkflows/nf-core/bed_scatter_bedtools/main.nf

```diff
@@ -0,0 +1,46 @@
+include { BEDTOOLS_SPLIT     } from '../../../modules/nf-core/bedtools/split/main'
+
+workflow BED_SCATTER_BEDTOOLS {
+
+    // IMPORTANT: Add the configuration in nextflow.config file to your modules.config
+
+    take:
+    ch_bed          // channel: [ meta, bed, scatter_count ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    ch_bed
+        .map(
+            { meta, bed, scatter_count ->
+                new_meta = meta.clone()
```

Comment:
is this doing a deep copy? We have had many issues in the past with shallow copies

Reply:
Fair point, I'll fix it

Reply:
Added!